$NetBSD: patch-configure,v 1.11 2024/07/15 21:14:50 schwarz Exp $

Disable the groff html check to prevent random PLIST variation.

--- configure.orig	2024-07-15 22:47:52.000000000 +0200
+++ configure	2024-07-15 22:48:11.000000000 +0200
@@ -51507,7 +51507,7 @@
 # Check for groff html support.
 { printf "%s\n" "$as_me:${as_lineno-$LINENO}: checking for HTML support in groff" >&5
 printf %s "checking for HTML support in groff... " >&6; }
-groff -t -mandoc -Thtml </dev/null >/dev/null
+false #groff -t -mandoc -Thtml </dev/null >/dev/null
 if test $? -ne 0 ; then
 	{ printf "%s\n" "$as_me:${as_lineno-$LINENO}: result: no" >&5
 printf "%s\n" "no" >&6; }
